src/staking-v3/components/Dapps.vue

```diff
@@ -88,7 +105,15 @@ export default defineComponent({
       return result;
     });
 
-    return { filteredDapps, getDappTier, getDappPageUrl };
+    const bg_img = {
```

Comment:
use camel case and more descriptive variable names. eg, `categoryBackgroundImages`

Reply:
@bobo-k2 fixed in [3e84327](https://github.com/AstarNetwork/astar-apps/pull/1132/commits/3e84327ff48fc254e74abb2b6277083f40a8386e)